### ISSUE TITLE
CLI issue

### ISSUE THREAD

**Issue by Bostonko** [IBM/db2-python#3]
Hello, I was trying to use your code but error CLI NOT INITIALIZED poped up. How can i INITIALIZED it before running subprocess? Thank you

**Comment by roger-e-sanders** [comment]
Do you have Db2 up and running in the background? If not, run the command db2start and then try again.

**Comment by Bostonko** [comment]
Hi Robert, Thank you for your response. I am trying to run db2 commands on
PC against the db2 database. Does your  response mean that i need to
manualy open the CLI and then run the script?

Thx

Dňa ut, 31. mar 2020, 22:51 Roger E. Sanders <notifications@github.com>
napísal(a):

**Comment by roger-e-sanders** [comment]
No. When you install Db2, you have the option to install the ODBC/CLI driver - I believe this is an option under "Developer tools". If this driver is installed, when you start the Db2 Database Manager instance that runs in the background (using the db2start command), the driver can be used by applications that make ODBC or CLI calls against the database. The Python driver is built on top of the ODBC/CLI library. 

In my earlier response, I forgot that when I was building the examples, I had to make sure I installed the ODBC/CLI library when I installed Db2. I believe there is a way to install the driver after Db2 has been installed, but I don't remember the process. You would need to look in the Db2 Knowledge Center to find that information.
